app/src/main/java/org/fossasia/susi/ai/adapters/recycleradapters/ChatFeedRecyclerAdapter.java

```diff
@@ -259,6 +264,8 @@ public int getItemViewType(int position) {
         if (item.getId() == -404) return DOTS;
         else if (item.getId() == -405) return NULL_HOLDER;
         else if (item.isDate()) return DATE_VIEW;
+        else if (item.getContent() != null && (item.getContent().endsWith(".jpg")||(item.getContent().endsWith(".png"))))
```

Comment:
Hm, this still is a hack.
Can we update the response object to contain a key which indicates what kind of data it's bringing?
That's a better way to handle this